packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -114,32 +113,34 @@ export default async function processDeployment({
         const missingSize = missing
           .map((sha: string) => total.get(sha).data.length)
           .reduce((a: number, b: number) => a + b, 0);
+        const totalSizeHuman = bytes.format(missingSize, { decimalPlaces: 1 });
 
-        output.stopSpinner();
-        bar = new Progress(`${chalk.gray('>')} Upload [:bar] :percent :etas`, {
+        const progressOpts = {
           width: 20,
           complete: '=',
-          incomplete: '',
-          total: missingSize,
-          clear: true,
-        });
-
-        bar.tick(0);
+          incomplete: '-',
```

Comment:
Perhaps instead of passing in the complete/incomplete characters, use the defaults defined in `progress()`. That will ensure we have better consistency when we go to add a progress bar elsewhere.

Reply:
In hindsight, maybe just remove complete/incomplete? Don’t even give devs the option to be inconsistent.